### PR TITLE
Updating The Graph 

### PR DESCRIPTION
**Pull request by idalithb**
Hi Linea! This includes updated docs for The Graph with a how-to guide for setting up a subgraph as quickly as possible.

### CODE REVIEW

File: docs/get-started/tooling/data-indexers/thegraph.mdx [hunk 1]
```diff
@@ -1,25 +1,280 @@
...
+powered by hundreds of independent Indexers, enabling your dapp to become truly
+decentralized.
 
-The Graph is a decentralized data indexer provider that indexes the Linea
-blockchain for you and exposes onchain data through an HTTPS API.
+Developers can use 100,000 free queries per month by using The Graph Network.
 
-We run The Graph indexers on Linea to allow you to leverage the power of this
-technology.
+## Quick Start
```
Comment (jlwllmr) [inline docs/get-started/tooling/data-indexers/thegraph.mdx:17]:
```suggestion
## Quickstart
```

File: docs/get-started/tooling/data-indexers/thegraph.mdx [hunk 2]
```diff
@@ -1,25 +1,280 @@
...
-TheGraph is now live with Linea Mainnet! For more information, take a look at
-the official [documentation](https://thegraph.com/docs/en/)
+1. Initialize your subgraph
+2. Publish your subgraph to The Graph Network
+3. Query from your dapp with your unique API key
 
-:::
+Here is a step-by-step walkthrough:
+
+## 1. Initialize Your Subgraph
```
Comment (jlwllmr) [inline docs/get-started/tooling/data-indexers/thegraph.mdx:29]:
```suggestion
## 1. Initialize your subgraph
```

File: docs/get-started/tooling/data-indexers/thegraph.mdx [hunk 3]
```diff
@@ -1,25 +1,280 @@
...
+
+### Create a Subgraph in Subgraph Studio
```
Comment (jlwllmr) [inline docs/get-started/tooling/data-indexers/thegraph.mdx:31]:
```suggestion
### Create a subgraph in subgraph studio
```

File: docs/get-started/tooling/data-indexers/thegraph.mdx [hunk 4]
```diff
@@ -1,25 +1,280 @@
...
+
+### Install the Graph CLI⁠
+
+On your local machine, run the following:
+
+```
+npm install -g @graphprotocol/graph-cli
+```
+
+### Initialize Your Subgraph⁠
```
Comment (jlwllmr) [inline docs/get-started/tooling/data-indexers/thegraph.mdx:67]:
```suggestion
### Initialize your subgraph⁠
```

File: docs/get-started/tooling/data-indexers/thegraph.mdx [hunk 5]
```diff
@@ -1,25 +1,280 @@
...
+      alt="alt text here"
+    />
+  </div>
+</div>
+
+Simply verify your contract on the block explorer, and the CLI will
+automatically obtain the ABI and set up your subgraph. The default settings will
+generate an entity for each event.
+
+## 2. Deploy & Publish
```
Comment (jlwllmr) [inline docs/get-started/tooling/data-indexers/thegraph.mdx:91]:
```suggestion
## 2. Deploy and publish
```

File: docs/get-started/tooling/data-indexers/thegraph.mdx [hunk 6]
```diff
@@ -1,25 +1,280 @@
...
+
+```bash
+$ graph auth <DEPLOY_KEY>
+$ graph deploy <SUBGRAPH_SLUG>
+```
+
+You will be asked for a version label. You can enter something like v0.0.1, but
+you're free to choose the format.
+
+### Test Your Subgraph
```
Comment (jlwllmr) [inline docs/get-started/tooling/data-indexers/thegraph.mdx:114]:
```suggestion
### Test your subgraph
```

File: docs/get-started/tooling/data-indexers/thegraph.mdx [hunk 7]
```diff
@@ -1,25 +1,280 @@
...
+<div class="center-container">
+  <div class="img-large">
+    <img
+      src="/img/get_started/tooling/data_indexers/the-graph/playground.png"
+      alt="alt text here"
+    />
+  </div>
+</div>
+
+### Publish Your Subgraph to The Graph's Decentralized Network
```
Comment (jlwllmr) [inline docs/get-started/tooling/data-indexers/thegraph.mdx:129]:
```suggestion
### Publish your subgraph to The Graph's decentralized network
```

**Review by jlwllmr — CHANGES_REQUESTED** [review]
This look great, thank you for the high-quality submission! 

Just a few pedantic suggestions to implement sentence case for headings.

**Comment by idalithb** [comment]
@jlwllmr Thank you so much for the feedback! Please let me know if anything else is needed to have this merged 😄

**Comment by jlwllmr** [comment]
@idalithb thanks! some of my suggestions haven't been committed yet, could you address those please?

**Comment by idalithb** [comment]
> @idalithb thanks! some of my suggestions haven't been committed yet, could you address those please?

Done ✅

**Review by jlwllmr — APPROVED** [review]
LGTM!

**Pull request merged**